actions/checkForAndPlaceOrder.ts

```diff
@@ -510,22 +524,43 @@ async function _placeOrder(
   return { result: PollResultCode.SUCCESS };
 }
 
-function _handleOrderBookError(
-  status: any,
-  data: any,
-  error: any
-):
-  | Omit<PollResultSuccess, "order" | "signature">
-  | PollResultTryNextBlock
-  | PollResultUnexpectedError {
+function _handleOrderBookError(params: {
+  status: any;
+  data: any;
+  error: any;
```

Comment:
So much `any` 🙈

Reply:
this comes from other PRs, i hope is better in the new standalone